### PR TITLE
Adds new flag to specify environment file to use

### PR DESCRIPTION
**Pull request by ladydascalie**
Implement #2

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -213,8 +215,18 @@ func applyEnvironment(input string) (output string, err error) {
 	return buf.String(), nil
 }
 
-func setEnvironmentFile(config string) error {
-	fullPath := filepath.Join(config, "env.toml")
+func setEnvironmentFile(config string, targetEnv string) error {
+	var envFile = "env.toml"
```
Comment (danmrichards) [inline main.go]: Doesn't look like this is ever modified, so probably should be `const envFile = "env.toml"` instead

**Comment by danmrichards** [comment]
Also conflicts (after merging #5 I guess)

**Comment by ladydascalie** [comment]
Yes, I expected the conflicts after the merge. They are now fixed!

**Comment by danmrichards** [comment]
B-e-a-utiful. Nice change @LadyDascalie, will make the tool that bit nicer to use.

**Pull request merged**